### PR TITLE
Fix problem with parsing dimension and media expressions

### PR DESCRIPTION
**Pull request by eduardoboucas**
SCSS:
```scss
@include media('>phone', '<tablet') {
  color: tomato;
}
```

Resulting CSS:
```css
@media (min-: 321px) and (max-: 767px) {
  color: tomato;
}
```

@HugoGiraudel this little fix seems to do it.

### CODE REVIEW

**Comment by eduardoboucas** [comment]
Fixes problem described above and also the one described in #45.

**Comment by HugoGiraudel** [comment]
Very odd as I didn't have the issue yesterday. I'll investigate.

**Comment by HugoGiraudel** [comment]
Actually it makes sense. I'll fix it in #45.

File: src/_config.scss [hunk 1]
```diff
@@ -34,14 +34,8 @@ $media-expressions: (
...
   'portrait': '(orientation: portrait)',
-  'retina2x': (
-    '(-webkit-min-device-pixel-ratio: 2)',
-    '(min-resolution: 192dpi)'
-  ), 
-  'retina3x': (
-    '(-webkit-min-device-pixel-ratio: 3)',
-    '(min-resolution: 350dpi)'
-  )
+  'retina2x': '(-webkit-min-device-pixel-ratio: 2) (min-resolution: 192dpi)',
```
Comment (HugoGiraudel) [inline src/_config.scss]: Are you sure this still works? Might be a problem. Test it.

File: src/helpers/_outputter.scss [hunk 2]
```diff
@@ -14,7 +14,7 @@
   @if length($queries) == 0 {
     @content;
   } @else {
-    @media #{inspect(nth($queries, 1))} {
+    @media #{inspect(unquote(nth($queries, 1)))} {
```
Comment (HugoGiraudel) [inline src/helpers/_outputter.scss]: This will fail.

**Comment by HugoGiraudel** [comment]
You can discard this and merge #48.

**Comment by eduardoboucas** [comment]
Fixed in #48.

**Pull request closed without merging**